corehq/apps/hqcase/management/commands/delete_cases.py

```diff
@@ -16,10 +16,6 @@ class Command(BaseCommand):
                     action='store_true',
                     dest='no_prompt',
                     help='Delete cases without prompting for confirmation'),
-        make_option('--by-last-submitter',
-                    action='store_true',
-                    dest='last_submitter',
-                    help='delete cases last updated by the specified user'),
```

Comment:
@esoergel I had to remove this option. wasn't sure how this was used, but seemed fine?